pkg/channel/consolidated/dispatcher/dispatcher.go

```diff
@@ -163,12 +164,13 @@ func NewDispatcher(ctx context.Context, args *KafkaDispatcherArgs) (*KafkaDispat
 type TopicFunc func(separator, namespace, name string) string
 
 type KafkaDispatcherArgs struct {
-	KnCEConnectionArgs *kncloudevents.ConnectionArgs
-	ClientID           string
-	Brokers            []string
-	KafkaAuthConfig    *client.KafkaAuthConfig
-	TopicFunc          TopicFunc
-	Logger             *zap.SugaredLogger
+	KnCEConnectionArgs       *kncloudevents.ConnectionArgs
```

Comment:
are these really used (via ConfigMap?)

Reply:
used ... or set (via CM?)

Reply:
Just checked. They're not used. Gonna remove them. Don't look like they would break anything.

Reply:
before removing: Please add a ticket, that we do that 

See recent/related change on IMC for these values. IMO we should use that here too (but different PR)...
/cc @slinkydeveloper

Reply:
Ok, sounds good.

Reply:
https://github.com/knative-sandbox/eventing-kafka/issues/312